examples/non_host_alignment_dag.json

```diff
@@ -0,0 +1,67 @@
+{
+  "output_dir_s3": "s3://idseq-samples-prod/samples/71/4542/results",
+  "targets": {
+    "host_filter_out": ["gsnap_filter_1.fa", "gsnap_filter_2.fa", "gsnap_filter_merged.fa"],
+    "gsnap_out": ["gsnap.m8", "gsnap.deduped.m8", "gsnap.hitsummary.tab", "gsnap_counts.json"],
+    "rapsearch2_out": ["rapsearch2.m8", "rapsearch2.deduped.m8", "rapsearch2.hitsummary.tab", "rapsearch2_counts.json"],
+    "taxon_count_out": ["web_sample.json"],
```

Comment:
Since we're renaming this... may I suggest we take this opportunity to use a more descriptive name? Such as taxon_counts.json

Reply:
sure.